packages/ingest/granule.js

```diff
@@ -152,14 +155,14 @@ class Granule {
    * Constructor for abstract Granule class
    *
    * @param {Object} buckets - s3 buckets available from config
-   * @param {Object} collection - collection configuration object
+   * @param {Object} context - rule context configuration object
```

Comment:
This is actually a critical change.  Collections are not passed to the workflows anymore since a workflow can process objects belonging to many collections.  The rule that triggers the workflow contain all optional parameters necessary for processing.